src/views/extended_search/ResultItem.vue

```diff
@@ -39,26 +39,26 @@
             </template>
             <template v-else-if="item.type === 'bigmapdiff'">
               <router-link class="serp-link" target="_blank"
-                :to="`/${item.body.network}/big_map/${item.body.ptr}`">
+                :to="sendTo(`/${item.body.network}/big_map/${item.body.ptr}`)">
                 <span class="hash text--primary">Big_map {{ item.body.ptr }}</span>
               </router-link>
               <span class="text--secondary" style="font-size: 20px;"> → </span>
               <router-link class="serp-link" target="_blank"
-                :to="`/${item.body.network}/big_map/${item.body.ptr}/${item.body.key_hash}`">
+                :to="sendTo(`/${item.body.network}/big_map/${item.body.ptr}/${item.body.key_hash}`)">
                 <span class="hash" v-html="highlight(item.body.key)"></span>
               </router-link>
             </template>
              <template v-else-if="item.type === 'token_metadata'">
               <span class="hash">Tokens</span>
               <span class="text--secondary" style="font-size: 20px;"> → </span>
-              <router-link class="serp-link" target="_blank" :to="`/${item.body.network}/${item.value}/tokens/?token_id=${item.body.token_id}`">
```

Comment:
Shouldn't be remove, no?

Reply:
@m-kus thanks, I guess, it's the aftermath of merge conflict, fixed